utils/hoursUtils.test.js

```diff
@@ -0,0 +1,131 @@
+jest.mock('airtable')
+const hoursUtils = require('./hoursUtils')
+
+const testHours = require('../testData/hours_response.json')
+
+
+const expectedHours = {
+    hours: [
+        {
+            day: 'sunday',
+            dayDigit: 0,
+            openTime: undefined,
+            closeTime: undefined,
+            isToday: false,
+            hoursSummary: 'Closed'
+        },
+        {
+            day: 'monday',
+            dayDigit: 1,
+            openTime: '11:00AM',
+            closeTime: '3:00PM',
+            isToday: false,
+            hoursSummary: '11:00AM - 3:00PM'
+        },
+        {
+            day: 'tuesday',
+            dayDigit: 2,
+            openTime: '12:00AM',
+            closeTime: '12:00AM',
+            isToday: false,
+            hoursSummary: 'Open 24 hours'
+        },
+        {
+            day: 'wednesday',
+            dayDigit: 3,
+            openTime: '12:00AM',
+            closeTime: '12:00AM',
+            isToday: false,
+            hoursSummary: 'Open 24 hours'
+        },
+        {
+            day: 'thursday',
+            dayDigit: 4,
+            openTime: '11:00AM',
+            closeTime: '3:00PM',
+            isToday: true,
+            hoursSummary: '11:00AM - 3:00PM'
+        },
+        {
+            day: 'friday',
+            dayDigit: 5,
+            openTime: undefined,
+            closeTime: undefined,
+            isToday: false,
+            hoursSummary: 'Closed'
+        },
+        {
+            day: 'saturday',
+            dayDigit: 6,
+            openTime: undefined,
+            closeTime: undefined,
+            isToday: false,
+            hoursSummary: 'Closed'
+        }
+    ]
+}
+
+describe('getHoursInfo', () => {
+    const periodsArray = [
+        "recmmC9oshIcu35c3",
+        "reczWnecVFbJ5GN4R",
+        "rec3nv0GVDNPQzs32",
+        "recuq0G9dcDh0fIFS"
+    ]
+
+    // ${"not open now"} | ${"has no hours today"} | ${'2021-02-26T16:01:58.135Z'} | ${false}   | ${undefined}    | ${undefined}
+    test.each`
```

Comment:
This structure for multiple test cases is awesome.  I've never seen it before, but it makes total sense.

Reply:
Yeeeees, parameterized tests are _THE BEST_